spec/requests/api/v1/merchants_request_spec.rb

```diff
@@ -38,14 +38,11 @@
     expect(merchant[:data][:attributes]).to have_key(:name)
   end
 
-  it "can create a new item" do
-
+  it "can create a new merchant" do
     headers = { "CONTENT_TYPE" => "application/json"}
-    post "/api/v1/merchants", :params => {
-      merchant: {
-        name: "Hello Kitty Kingdom"
-        }
-      }
+    merchant_params = { name: "Toy Plane Emporium" }
+
+    post "/api/v1/merchants", params: JSON.generate(merchant_params), headers: headers
```

Comment:
Important to wrap the params in JSON data to ensure that they are coming in correctly to be processed and rendered!@